crates/rari-tools/src/sidebars.rs

```diff
@@ -433,7 +433,7 @@ mod test {
                 Box_alignment_in_block_layout: CSS first steps
                 Box_model: Box model
               fr:
-                Backgrounds_and_Borders: Tutoriels
+                Backgrounds_and_Borders: Tutorials
```

Comment:
```suggestion
                Backgrounds_and_Borders: Tutoriels
```

Reply:
@bsmth I cannot push to your branch, because I opened the PR.

Reply:
Ah, good catch